### PR TITLE
Fix issue #58 in alerting-plugin, enhance trigger execution exception messsage

### PR DESCRIPTION
**Pull request by seraphjiang**
this is ux part of code change to resolve alerting-plugin 58. 
it is target to ship with open distro 1.1.0

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

**Comment by seraphjiang** [comment]
Test Suites: 6 skipped, 70 passed, 70 of 76 total
Tests:       20 skipped, 341 passed, 361 total
Snapshots:   121 passed, 121 total
Time:        40.986s
Ran all test suites.
✨  Done in 42.35s.

File: package.json [hunk 1]
```diff
@@ -1,12 +1,12 @@
 {
   "name": "opendistro-alerting",
-  "version": "1.0.0.0",
+  "version": "1.1.0.0",
```
Comment (mihirsoni) [inline package.json]: Can you remove these changes from this PR.

Comment (seraphjiang) [reply package.json]: sure

File: public/pages/CreateTrigger/components/TriggerQuery/TriggerQuery.js [hunk 2]
```diff
@@ -113,17 +112,22 @@ const TriggerQuery = ({
             )}
           />
         </EuiFlexItem>
+        <EuiFlexItem grow={false}>
+          <EuiFormRow label="Trigger condition response:" fullWidth>
+            <EuiCodeEditor
+              theme={isDarkMode ? 'sense-dark' : 'github'}
+              width="100%"
+              height="200px"
+              value={getExecuteMessage(executeResponse)}
```
Comment (ylwu-amzn) [inline public/pages/CreateTrigger/components/TriggerQuery/TriggerQuery.js]: If trigger run successfully, will show 'true' to user, can we change to a more readable message? Such as "Trigger ran successfully"

File: public/pages/CreateTrigger/components/TriggerQuery/TriggerQuery.js [hunk 3]
```diff
@@ -113,17 +112,22 @@ const TriggerQuery = ({
             )}
           />
         </EuiFlexItem>
+        <EuiFlexItem grow={false}>
+          <EuiFormRow label="Trigger condition response:" fullWidth>
+            <EuiCodeEditor
+              theme={isDarkMode ? 'sense-dark' : 'github'}
```
Comment (ylwu-amzn) [inline public/pages/CreateTrigger/components/TriggerQuery/TriggerQuery.js]: I think EUI can handle the theme by default. Can you confirm if it works for all browsers if remove this line?

Comment (mihirsoni) [reply public/pages/CreateTrigger/components/TriggerQuery/TriggerQuery.js]: CodeEditor doesn't handle it so we need to handle it ourself.

File: public/pages/CreateTrigger/components/TriggerQuery/TriggerQuery.js (hunk 2 above)
Comment (seraphjiang) [reply public/pages/CreateTrigger/components/TriggerQuery/TriggerQuery.js]: file a separate issue to track suggestion.

https://github.com/opendistro-for-elasticsearch/alerting-kibana-plugin/issues/62

**Pull request merged**